api-service/service/hosts.go

```diff
@@ -254,7 +256,22 @@ func (as *APIService) GetHost(hostname string, olderThan time.Time, raw bool) (*
 						count *= float64(model.GetFactorByMetric(licType.Metric))
 						host.Features.Oracle.Database.Databases[i].Licenses[j].Count = count
 					}
+
+					if lic.Name == "Real Application Clusters" && lic.Count > 0 {
+						realApplicationClusters = true
+					}
+
+					if realApplicationClusters && lic.Name == "Real Application Clusters One Node" && lic.Count > 0 {
+						indexList = append(indexList, j)
+					}
+				}
+			}
+			if len(indexList) > 0 {
+				for _, i3 := range indexList {
```

Comment:
Please, don't use i3!
If `i` has already been taken, you can go for `j`, `k`, `l`...